### PR TITLE
Simplify image management + Add Far Manager

### PR DESCRIPTION
**Pull request by Koubek**
- This is "how-to" simplify a little bit the image management (image
name, version no., tagging).
- Added Far Manager as an example how to implement those changes. This
image generally has no use when deployed independently but Far Manager
could be deployed within another image/container to simplify some
operations. I can see it quite useful in the development phase to move
some files easily between two folders, or edit someone (of course, you
can use PSEdit when working with PowerShell ISE).

### CODE REVIEW

**Comment by Koubek** [comment]
Just one more note: the `_push.bat` has both `docker push` lines blocked.

File: farmanager/_build.bat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+call _presetvars.bat
```
Comment (StefanScherer) [inline farmanager/_build.bat]: I have switched the build scripts from bat to ps1.

File: farmanager/_run.demo.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+call _presetvars.bat
```
Comment (StefanScherer) [inline farmanager/_run.demo.bat]: This script may be transfered to  `test.ps1` to have a simple test in AppVeyor build.

File: farmanager/__imageversion.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+3.0.4900
```
Comment (StefanScherer) [inline farmanager/__imageversion.txt]: The version tags could be "grepped" from the Dockerfile using eg. `ENV FAR_VERSION 3.0.4900` inside the Dockerfile. I also have to improve that for other build scripts (eg. in node folder) to reduce the number of duplicate version strings.

File: farmanager/_push.bat [hunk 4]
```diff
@@ -0,0 +1,7 @@
+call _presetvars.bat
```
Comment (StefanScherer) [inline farmanager/_push.bat]: this should be transfered to `push.ps1`

**Comment by Koubek** [comment]
Please, consider this just as a proposal, nothing more. I use the file structure to copy it for each "project" I create. I can copy those files and change just the image name, image version and of course, Dockerfile and the rest of the project related stuff. But the build process remains, the tagging as well. I do not need to bother with it again and again (except some specific projects).

File: farmanager/__imageversion.txt (hunk 3 above)
Comment (Koubek) [inline farmanager/__imageversion.txt]: This is really good idea but in relation to my previous response I would not use a project related tag (environment variable name) to keep the rest of the files (build, tag, test etc.) generic as much as possible.

**Comment by StefanScherer** [comment]
I still like the idea, but don‘t want to add new files. The version can be grepped from Dockerfile like it https://github.com/StefanScherer/dockerfiles-windows/blob/master/traefik/push.ps1

The repo name should be grepped from some git remote command or something similar. I use build.ps1 scripts so we have PowerShell which can do the task.

**Pull request closed without merging**